x-pack/plugins/lens/common/expressions/metric_chart/metric_chart.ts

```diff
@@ -38,6 +38,24 @@ export const metricChart: ExpressionFunctionDefinition<
         defaultMessage: 'The chart title.',
       }),
     },
+    titleSize: {
+      types: ['string'],
+      help: i18n.translate('xpack.lens.metric.titleSize.help', {
+        defaultMessage: 'The chart title size.',
+      }),
+    },
```

Comment:
From the CSS we are not just changing the size of the label but also the size of the value. We should clearly describe that here